spec/rails_helper.rb

```diff
@@ -39,6 +39,10 @@
   end
 
   config.before(:each) do
+    allow(I18n).to receive(:locale).and_return(:en)
```

Comment:
I don't think this should be an `allow`, IMO we should just set the locale each time. As written, during a test run, something could set the locale to something other than `:en` but it would always be read back as `:en` which is wrong

```
I18n.locale = :en
```